iidm/iidm-converter-api/src/main/java/com/powsybl/iidm/export/ExportOptions.java

```diff
@@ -153,6 +152,20 @@ public ExportOptions setVersion(String version) {
         return this;
     }
 
+    public ExportOptions addExtensionVersion(String extensionName, String extensionVersion) {
```

Comment:
Could you add documentation about the usage of this method. I'm sorry but I never remember how and when to use this